From be849381c7c7cbf8ea224d56696a6aa4829e2a8f Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 25 Sep 2024 11:25:05 +0800
Subject: [PATCH 2/3] services4: srvkm: osfunc: drop unused parameter for
 get_user_pages()

kernel commit 54d020692b342 (mm/gup: remove unused
vmas parameter from get_user_pages()) introduced new API for
from get_user_pages(), drop unused parameter for get_user_pages()
in order to compatible kernel v6.6

(LOCAL REV: NOT UPSTREAM) - Wrlinux specific

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 services4/srvkm/env/linux/osfunc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services4/srvkm/env/linux/osfunc.c b/services4/srvkm/env/linux/osfunc.c
index de0e7ae..7cc2c91 100644
--- a/services4/srvkm/env/linux/osfunc.c
+++ b/services4/srvkm/env/linux/osfunc.c
@@ -3622,7 +3622,7 @@ PVRSRV_ERROR OSAcquirePhysPageAddr(IMG_VOID *pvCPUVAddr,
 
     /* Get page list */
     psInfo->iNumPagesMapped = get_user_pages(
-		uStartAddr, psInfo->iNumPages, 1, psInfo->ppsPages, NULL);
+		uStartAddr, psInfo->iNumPages, 1, psInfo->ppsPages);
 
     if (psInfo->iNumPagesMapped >= 0)
     {
-- 
2.34.1

